extensions/amp-subscriptions/amp-subscriptions.md

```diff
@@ -47,160 +74,366 @@ Because of standardization of markup, support for multiple providers, and improv
+To assist access services and use cases, AMP Access introduced the concept of _Reader ID_.
 
-## Product configuration
+The Reader ID is an anonymous and unique ID created by the AMP ecosystem. It is unique for each Reader/Publisher pair - a Reader is identified differently to two different Publishers. It is a non-reversible ID. The Reader ID is included in all AMP/Publisher communications and has very high entropy. Publishers can use the Reader ID to identify the Reader and map it to their own identity systems.
 
-`amp-subscriptions` relies on the Schema.org page-level configuration for two main properties:
+The Reader ID is constructed on the user device and intended to be long-lived. However, it follows the normal browser storage rules, including those for incognito windows. The intended lifecycle of a Reader ID is 1 year between uses or until the user clears their cookies. The Reader IDs are not currently shared between devices.
 
-1.  The product ID that the user must be granted to view the content.
-2.  Whether this content requires this product at this time.
+The Reader ID is constructed similarly to the mechanism used to build ExternalCID described [here](https://docs.google.com/document/d/1f7z3X2GM_ASb3ZCI_7tngglxwS6WoWi1EB3aKzdf6vo/edit#heading=h.hb9q0wpwwhuf). An example Reader ID is `amp-OFsqR4pPKynymPyMmplPNMvxSTsNQob3TnK-oE3nwVT0clORaZ1rkeEz8xej-vV6`.
 
-The JSON-LD and Microdata formats are supported.
+### Configuration
 
-## JSON-LD markup
+The `amp-subscriptions` extension must be configured using JSON configuration:
 
-Using JSON-LD, the markup would look like:
+<table>
+  <tr>
+    <th>Property</th>
+    <th>Values</th>
+    <th>Description</th>
+  </tr>
+  <tr>
+    <td class="col-fourty"><code>services</code></td>
+    <td>&lt;array&gt; of &lt;object&gt;</td>
+    <td>This <code>array</code> must include:<ul><li>One <a href="#local-service">Local Service</a></li><li>Zero or more <a href="#vendor-services">Vendor Services</a>.</li></ul></td>
+  </tr>
+  <tr>
+    <td class="col-fourty"><code>score</code></td>
+    <td>&lt;object&gt;</td>
+    <td>Determines which service is selected if no valid entitlements are returned.<br/>See <a href="#service-score-factors">Service Score Factors</a> for more details.</td>
+  </tr>
+  <tr>
+    <td class="col-fourty"><code>fallbackEntitlement</code></td>
+    <td>&lt;object&gt;</td>
+    <td>Determines what level of access the Reader should have if all services fail to respond to the Authorization requests.<br/>See <a href="#fallback-entitlement">Fallback Entitlement</a> for more details.</td>
+  </tr>
+</table>
+
+Here is an example of a configuration:
 
 ```html
-<script type="application/ld+json">
+<script type="application/json" id="amp-subscriptions">
```

Comment:
does the id field need to point to `amp-subscriptions` or the id of the the amp-subscriptions component?

Reply:
It needs to point to `amp-subscriptions` as this is how the runtime knows which specific configuration is for `amp-subscriptions` vs. other components.